frontend/src/app/patients/Components/PersonForm.tsx

```diff
@@ -35,6 +40,7 @@ interface Props {
     formChanged: boolean
   ) => React.ReactNode;
   getFooter: (onSave: () => void, formChanged: boolean) => React.ReactNode;
+  isPatientView?: boolean;
```

Comment:
Sidebar, though this suggestion isn't for this PR: I prefer this more expressive prop name to the `isModal` that we're using in AoEForm, and I think we should update AoEForm accordingly